src/pages/home/report/ReactionList/BaseReactionList.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {View, FlatList} from 'react-native';
+import PropTypes from 'prop-types';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import ReactionListItem from './ReactionListItem';
+import styles from '../../../../styles/styles';
+import HeaderReactionList from './HeaderReactionList';
+import CONST from '../../../../CONST';
+import participantPropTypes from '../../../../components/participantPropTypes';
+import {
+    propTypes as reactionPropTypes,
+    defaultProps as reactionDefaultProps,
+} from './reactionPropTypes';
+
+const propTypes = {
+
+    /**
+    *  Array of personal detail objects
+    */
+    users: PropTypes.arrayOf(participantPropTypes),
+
+    /**
+     * Returns true if the current account has reacted to the report action (with the given skin tone).
+     */
+    hasUserReacted: PropTypes.bool,
+
+    ...reactionPropTypes,
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    ...reactionDefaultProps,
+};
+
+const renderItem = ({item}) => <ReactionListItem item={item} />;
+const keyExtractor = (item, index) => `${item.login}+${index}`;
+const getItemLayout = (_, index) => ({
```

Comment:
If the linter does not complain its not a problem for me 🤣 
But besides that, I'm not too picky about docs. It would be nice if param docs are added here since it will help with readability but i won't block the PR on it.